src/app/pages/search/decision-support-interventions/decision-support-interventions-view.jsx

```diff
@@ -0,0 +1,307 @@
+  },
+  pageBody: {
+    display: 'grid',
+    gridTemplateColumns: ' 1fr',
+    gap: '16px',
+    padding: '16px 32px',
+    backgroundColor: '#f9f9f9',
+    [theme.breakpoints.up('md')]: {
+      gridTemplateColumns: '2fr 1fr',
+    },
+  },
+  pageContent: {
+    display: 'grid',
+    gridTemplateRows: '3fr 1fr',
+  },
+  stickyColumn: {
+    position: 'sticky',
+    left: 0,
+    boxShadow: 'rgba(149, 157, 165, 0.1) 0px 4px 8px',
+    backgroundColor: '#ffffff',
+    overflowWrap: 'anywhere',
+    [theme.breakpoints.up('sm')]: {
+      minWidth: '275px',
+    },
+  },
+  tableContainer: {
+    overflowWrap: 'normal',
+    border: '.5px solid #c2c6ca',
+    margin: '0px 32px',
+    width: 'auto',
+  },
+  tableResultsHeaderContainer: {
+    display: 'grid',
+    gap: '8px',
+    margin: '16px 32px',
+    gridTemplateColumns: '1fr',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    [theme.breakpoints.up('sm')]: {
+      gridTemplateColumns: 'auto auto',
+    },
+  },
+  resultsContainer: {
+    display: 'grid',
+    gap: '8px',
+    justifyContent: 'start',
+    gridTemplateColumns: 'auto auto',
+    alignItems: 'center',
+  },
+  wrap: {
+    flexFlow: 'wrap',
+  },
+});
+
+/* eslint object-curly-newline: ["error", { "minProperties": 5, "consistent": true }] */
+const headers = [
+  { property: 'chpl_id', text: 'CHPL ID', sortable: true },
+  { property: 'developer', text: 'Developer', sortable: true },
+  { property: 'product', text: 'Product', sortable: true },
+  { property: 'version', text: 'Version', sortable: true },
+  { text: 'Status', extra: <ChplCertificationStatusLegend /> },
+  { text: 'Risk Management Summary Information' },
+  { text: 'Actions', invisible: true },
+];
+
+function ChplDecisionSupportInterventionsSearchView() {
+  const storageKey = 'storageKey-decisionSupportInterventionsView';
+  const { analytics } = useAnalyticsContext();
+  const [listings, setListings] = useState([]);
+  const [orderBy, setOrderBy] = useStorage(`${storageKey}-orderBy`, 'developer');
+  const [pageNumber, setPageNumber] = useStorage(`${storageKey}-pageNumber`, 0);
+  const [pageSize, setPageSize] = useStorage(`${storageKey}-pageSize`, 25);
+  const [sortDescending, setSortDescending] = useStorage(`${storageKey}-sortDescending`, false);
+  const [recordCount, setRecordCount] = useState(0);
+  const classes = useStyles();
+  const toggledCsvDefaults = ['riskManagementSummaryInformation'];
+
+  const filterContext = useFilterContext();
+  const { data, isError, isLoading } = useFetchListings({
+    orderBy,
+    pageNumber,
+    pageSize,
+    sortDescending,
+    query: filterContext.queryString(),
+  });
+  
+  useEffect(() => {
+    if (isLoading) { return; }
+    if (isError || !data.results) {
+      setListings([]);
+      return;
+    }
+    setListings(data.results.map((listing) => ({
+      ...listing,
+      riskManagementSummaryInformationValue: listing.riskManagementSummaryInformation?.value || '',
```

Comment:
There is some sort of default value we have on Listing Details pages for listings with b11 that do not have this URL, and that value should show in this table as well as in the download file. There was a lot of discussion about what that phrase should be, and ASTP has mentioned wanting it in calls as recently as this past Tuesday